src/components/ResourceCard/ResourceCard.js

```diff
@@ -123,18 +128,21 @@ ResourceCard.propTypes = {
   resources: shape({}).isRequired,
   patientAgeAtResources: shape({}).isRequired,
   selectedCollectionId: string.isRequired,
-  collectionResourceIds: shape({}),
+  collectionsResourceIds: shape({}).isRequired,
+  previewCollectionId: string.isRequired,
+  isCollectionPreview: bool,
```

Comment:
are all these new things being used to compute `inCollection` ?  

...if so:  there could be a simpler approach, either manually prop-drilling the context, and/or: a new selector that uses `ownProps` ?

Reply:
oh, `isCollectionPreview` could be that prop-drilled context?

so: a new selector uses `collectionsResourceIds`, `previewCollectionId` (internally) and `resourceId` and `isCollectionPreview` (from `ownProps`) to compute `inCollection`? 


...another, completely different alternative might be to build `inCollection` into the shape of the view-model that is being iterated over to draw each type / subtype